ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetTapes.java

```diff
@@ -15,23 +15,40 @@
 
 package com.spectralogic.ds3cli.command;
 
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3cli.Arguments;
 import com.spectralogic.ds3cli.View;
 import com.spectralogic.ds3cli.ViewType;
 import com.spectralogic.ds3cli.models.GetTapesResult;
+import com.spectralogic.ds3client.utils.Guard;
 import com.spectralogic.ds3cli.views.cli.GetTapesView;
 import com.spectralogic.ds3client.commands.spectrads3.GetTapesSpectraS3Request;
 import com.spectralogic.ds3client.commands.spectrads3.GetTapesSpectraS3Response;
+import org.apache.commons.cli.Option;
+
+import static com.spectralogic.ds3cli.ArgumentFactory.BUCKET;
 
 public class GetTapes extends CliCommand<GetTapesResult> {
+    private final GetTapesSpectraS3Request getTapesSpectraS3Request = new GetTapesSpectraS3Request();
+    private final static ImmutableList<Option> optionalArgs = ImmutableList.of(BUCKET);
```

Comment:
Our standard naming convention for final statics is to declare them with all caps using underscores to separate words.